src/storage/index/LookUpIndexProcessor.cpp

```diff
@@ -59,8 +61,16 @@ void LookUpEdgeIndexProcessor::process(const cpp2::LookUpIndexRequest& req) {
         }
         s.set_columns(std::move(cols));
         this->resp_.set_schema(std::move(s));
+    } else {
+        LOG(ERROR) << "schema is null";
     }
```

Comment:
I think this is redundant. If schema is null means the yield clause is null , There is no need to return schema . This is not a error.

Reply:
👍